packages/data-transformation/fhir-to-csv/server.py

```
@@ -0,0 +1,88 @@
+
+@app.errorhandler(404)
+def not_found(error):
+    return jsonify({"error": "Endpoint not found"}), 404
+
+@app.errorhandler(500)
+def internal_error(error):
+    return jsonify({"error": "Internal server error"}), 500
+
+if __name__ == '__main__':
+    port = int(os.environ.get('PORT', 8000))
+    host = os.environ.get('HOST', '0.0.0.0')
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)